CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-[cygnus-common][SQLBackendImpl] Exception handle for connection/statement SQL objects.
```

Comment:
This line should be deleted?

Reply:
That entry is about previous PR https://github.com/telefonicaid/fiware-cygnus/issues/1907, but is not deleted, just edited